tests/test_core.py

```diff
@@ -324,20 +324,20 @@ def test_scalar_dtype() -> None:
     assert c.dtype is None
 
 
-def test_scalar_pickle(daa: Array) -> None:
-    import pickle
-
-    s = 2
-    c1 = new_known_scalar(s)
-    s_dumped = pickle.dumps(c1)
-    c2 = pickle.loads(s_dumped)
-    assert_eq(c1, c2)
-    s1 = dak.sum(daa["points"]["y"], axis=None)
-    s_dumped = pickle.dumps(s1)
-    s2 = pickle.loads(s_dumped)
-    assert_eq(s1, s2)
-
-    assert s1.known_value is None
+# def test_scalar_pickle(daa: Array) -> None:
```

Comment:
Why does this fail? Can it be reinstated? If not, we should either comment or remove it. I think I had a snippet to re-add the _meta objects to the layers, if that's what we need.

Reply:
Yes that snippet will resurrect that test

Reply:
Do you need me to apply it?

Reply:
I am presently in a car so it would be appreciated :-)